### ISSUE TITLE
[Bug]: MACOS - COMMAND + L triggers skip forward in addition to focusing address bar

### ISSUE THREAD

**Issue by lewgrant** [FreeTubeApp/FreeTube#9242]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads). **The only official downloads are from the [https://freetubeapp.io](https://freetubeapp.io/#download) website, the [FreeTubeApp/FreeTube GitHub repository](https://github.com/FreeTubeApp/FreeTube/releases) and the [io.freetubeapp.FreeTube flatpak](https://flathub.org/en/apps/io.freetubeapp.FreeTube) from Flathub.**
- [x] I have [searched the issue tracker for **open and closed** issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

Reproduce this bug by:

1. playing a video
2. while video is still playing press `command` + `L` to focus/select address bar
3. Note that while the address bar is focused/selected as expected the video also skips forward several seconds as if you had pressed `L` alone.

### Expected Behavior

The use of `command` + `L` should focus/select address bar without effecting playback.

The `L` keyboard shortcut should not be triggered when modifier keys are pressed.

This issue should be fixed for the benefit of keyboard users and accessibility.

Many thanks in advance for an app I use daily and highly recommend to anyone that will listen.

### Issue Labels

accessibility issue, keyboard control not working, usability issue

### FreeTube Version

0.24.0

### Operating System Version

Mac OS Tahoe 26.5.1

### Installation Method

.dmg

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest **official** [nightly build](https://docs.freetubeapp.io/development/nightly-builds), do not check this box if you are using an unofficial or custom build.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE can you repro this?

**Comment by PikachuEXE** [comment]
Ya I can

**Comment by PikachuEXE** [comment]
https://github.com/FreeTubeApp/FreeTube/pull/9255
